addons/website_customer/controllers/main.py

```diff
@@ -48,6 +48,9 @@ def customers(self, country=None, industry=None, page=0, **post):
         search_value = post.get('search')
 
         domain = [('website_published', '=', True), ('assigned_partner_id', '!=', False)]
+        countries = Partner.sudo().read_group(domain, ["id", "country_id"], groupby="country_id", orderby="country_id")
+        industries = Partner.sudo().read_group(domain, ["id", "industry_id"], groupby="industry_id", orderby="industry_id")
```

Comment:
Actually the python change here seems to be more reasons to not have to do this in stable. From what I understand, the dropdowns were always shown before but they were actually empty anyway since countries/industries etc were searched sing the "search" value too? So you are actually not fixing something but changing behaviors, no?